src/uebung_3_3/BMI.java

```diff
@@ -0,0 +1,74 @@
+package uebung_3_3;
+
+public class BMI {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+
+		double Alter = 19; // in Jahren
+		double K�rpergewicht = 55; // in kg
+		double K�rpergr��e = 1.70; // in m
+		double Geschlecht = 1; // weiblich = 0; m�nnlich = 1

```

Comment:
double für eine binäre Information ist ein bisschen "Overkill". Einfacher wäre es mit einem Boolean, und auch leichter zu verstehen in Bedingungen z.B. if (istMaennlich) bzw. !istMaennlich